proposals/0251-font-styles.md

```diff
@@ -28,16 +28,22 @@ The HMI should be able to provide a list of supported hyper text elements to SDL
 ```xml
 <struct name="TextField" since="1.0">
 :
-    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minvalue="1" maxvalue="100" since="6.x">    
+    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minsize="1" maxsize="100" since="6.x">    
     </param>
 </struct>
 ```
 
 The HMI API equivalent should exclude the versioning attributes.
 
+The head unit would just send an array of html tags without the < > characters.
```

Comment:
I would recommend pointing to this link for currently available elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Inline_text_semantics